web/leaderboard/views.py

```diff
@@ -18,17 +19,43 @@ def index(request):
         )
         env_obj = env_obj_by_id[env_id]
         env_dict[env_obj] = [run_obj_by_id[run_id] for run_id in runs]
-    print("env_dict is:", dict(env_dict))
     context = {"env_dict": dict(env_dict), "is_debug": settings.DEBUG}
     return render(request, "leaderboard/index.html", context)
 
 
+def run_list(request):
+    agent_runs = Run.objects.filter(
+        data__tags__contains={"pcs.is_agent_run": "True"}
+    )
+    component_runs = Run.objects.filter(
+        data__tags__contains={"pcs.is_component_run": "True"}
+    )
+    component_runs_with_starttimes = {}
+    for run in component_runs:
```

Comment:
I think a more canonical way to achieve this is to put something like the following on the Run model:

```python
@property
def start_time(self):
    start_time_str = self.info["start_time"]
    start_time = datetime.fromtimestamp(int(start_time_str) / 1000)
    return start_time.strftime("%d/%m/%y %H:%m")
```

This, however, can end up being somewhat confusing because you can use `run.start_time` in templates, but it's not actually a column in the DB, and so the ORM will not be happy if you try to filter or sort using `start_time` (even though it's indistinguishable from other columns when you're using it on a `Run` object).

I think either way is ok for now, but we should probably move it onto the `Run` object if we end up wanting to use this in more places.